google-cloud-bom/pom.xml

```diff
@@ -133,12 +133,12 @@
     <generated-proto-beta.version>0.1.28</generated-proto-beta.version>
     <generated-proto-ga.version>1.0.4</generated-proto-ga.version>
     <google.auth.version>0.9.0</google.auth.version>
-    <grpc.version>1.7.0</grpc.version>
-    <nettyssl.version>2.0.6.Final</nettyssl.version>
+    <grpc.version>1.9.0</grpc.version>
+    <nettyssl.version>2.0.7.Final</nettyssl.version>
     <guava.version>20.0</guava.version>
     <http-client.version>1.23.0</http-client.version>
-    <protobuf.version>3.4.0</protobuf.version>
-    <opencensus.version>0.9.1</opencensus.version>
+    <protobuf.version>3.5.1</protobuf.version>
```

Comment:
Note, artman is using protoc 3.4 (3.5.1 is backward compatible, right?)

Reply:
There are no known compatibility issues.